.update/do.py

```diff
@@ -0,0 +1,130 @@
+import click
+import requests
+from pathlib import Path
+
+from cnxcommon import ident_hash
+
+
+here = Path(__file__).parent
+CNX_HOST = 'archive-staging.cnx.org'
+MAP_FILEPATH = here.resolve().parent / 'roles/webview/files/etc/nginx/uri-maps/rex-uris.map'
+
+
+def get_rex_release_json_url(host):
+    return f'https://{host}/rex/release.json'
+
+
+def get_book_slug(book_id):
+    url = (
```

Comment:
This inline var seems dangerous—like it could change and silently fail; by "it" I mean the URL's accepted params or the path.

Reply:
IMHO, all of this code is short-lived. So it's scrap that we are pulling together to build scaffolding for the larger more important project. This url is only now changing. Technically, it's a versioned API, so therefore it shouldn't change.

Reply:
I agree with @pumazi comment. Because it is shortlived it's not a super big issue. I mean you can make url params super save but this should be only necessary for very dynamic content or when an id can be modified from outside (attack vector).